File: app/src/main/java/wxm/KeepAccount/ui/help/ACHelp.java
package wxm.KeepAccount.ui.help;

import android.os.Bundle;

import wxm.androidutil.Switcher.ACSwitcherActivity;


/**
 * UI for help
 */
public class ACHelp extends ACSwitcherActivity<FrgHelp> {
    public static final String STR_HELP_TYPE = "HELP_TYPE";

    public static final String STR_HELP_MAIN = "help_main";
    public static final String STR_HELP_START = "help_start";

    @Override
    protected void setupFragment(Bundle bundle) {
        addFragment(new FrgHelp());
    }
}

File: app/src/main/java/wxm/KeepAccount/ui/help/ACHelp.kt
package wxm.KeepAccount.ui.help

import android.os.Bundle
import wxm.KeepAccount.ui.base.ACBase.ACBase

/**
 * UI for help
 */
class ACHelp : ACBase<FrgHelp>() {

    override fun setupFragment(bundle: Bundle?) {
        addFragment(FrgHelp())
    }

    companion object {
        const val STR_HELP_TYPE = "HELP_TYPE"

        const val STR_HELP_MAIN = "help_main"
        const val STR_HELP_START = "help_start"
    }
}
